src/theory/quantifiers/ceg_instantiator.h

```diff
@@ -41,14 +41,24 @@ class CegqiOutput {
 class Instantiator;
 
 
+class TermProperties {
+public:
+  TermProperties() : d_type(-1) {}
+  int d_type;
+  // for arithmetic
```

Comment:
I am not sure I understand the purpose of this comment. Could you make it more detailed (maybe not needed if there is a comment for the class in general as suggested above)?

Reply:
See general comment [1] below.